packages/react-relay/__tests__/__generated__/RelayClient3DModuleTestQuery.graphql.js

```diff
@@ -113,7 +113,7 @@ return {
                         "fragmentPropName": "data",
                         "kind": "ModuleImport",
                         "componentModuleProvider": () => require('./../ClientUser.react'),
-                        "operationModuleProvider": () => require('react-relay/__tests__/__generated__/./RelayClient3DModuleTestFragmentClientUser_data$normalization.graphql')
+                        "operationModuleProvider": () => require('packages/react-relay/__tests__/__generated__/./RelayClient3DModuleTestFragmentClientUser_data$normalization.graphql')
```

Comment:
This is unrelated to my changes (you can see the compiler output check failing on main).